qiskit_experiments/calibration_management/backend_calibrations.py

```diff
@@ -54,39 +44,58 @@ class BackendCalibrations(Calibrations):
+        backend: Optional[Backend] = None,
+        library: Optional[BasisGateLibrary] = None,
+        add_parameter_defaults: bool = True,
+        control_config: Optional[Dict[Tuple[int, ...], List[ControlChannel]]] = None,
+        coupling_map: Optional[List[List[int]]] = None,
         num_qubits: Optional[int] = None,
     ):
         """Setup an instance to manage the calibrations of a backend.
 
-        BackendCalibrations can be initialized from a basis gate library, i.e. a subclass of
-        :class:`BasisGateLibrary`. As example consider the following code:
-
-        .. code-block:: python
-
-            cals = BackendCalibrations(
-                    backend,
-                    library=FixedFrequencyTransmon(
-                        basis_gates=["x", "sx"],
-                        default_values={duration: 320}
-                    )
-                )
-
         Args:
             backend: A backend instance from which to extract the qubit and readout frequencies
                 (which will be added as first guesses for the corresponding parameters) as well
-                as the coupling map.
+                as the coupling map. If the backend is not given then the calibrations will be
+                initialized with the following key-word arguments: control_config, coupling_map,
+                num_qubits.
             library: A library class that will be instantiated with the library options to then
                 get template schedules to register as well as default parameter values.
-            num_qubits: Number of qubits in case the backend object fails to specify this in its
-                configuration.
+            add_parameter_defaults: A boolean to indicate whether the default parameter values of
+                the given library should be used to populate the calibrations. By default this
+                value is True but can be set to false when deserializing a calibrations object.
+            control_config: A configuration dictionary of any control channels. The
+                keys are tuples of qubits and the values are a list of ControlChannels
+                that correspond to the qubits in the keys. This argument is optional and only
+                needed if the backend is not provided.
+            coupling_map: The coupling map of the device. This option is not needed if the backend
+                is provided.
+            num_qubits: The number of qubits of the system. This parameter is only needed if the
+                backend is not given.
 
         Raises:
-            CalibrationError: If the backend configuration does not have num_qubits and num_qubits
-                is None.
+            CalibrationError: if backend is None and any of control_config, coupling_map, num_qubits
+                are None.
         """
-        super().__init__(getattr(backend.configuration(), "control_channels", None))
+        if backend is None:
```

Comment:
Seems like 
```python
class BackendCalibrations(
    add_parameter_defaults,
    control_config,
    num_qubits,
    library=None,
):
    @classmethod
    def from_backend(backend, library=None):
        ...
```
is cleaner? though this is breaking change.

Reply:
Done in this commit: 7255df2. I think that we should not be afraid of breaking changes. I also took this as an opportunity to merge `BackendCalibrations` into `Calibrations`, something that we have been mentioning for a bit.